### PR TITLE
corrected structure

### PR DESCRIPTION
**Pull request by LeonardBesseau**
Removed duplicate files

### CODE REVIEW

File: Server/routes/elementRoutes.js [hunk 1]
```diff
@@ -32,6 +29,7 @@ router.post('/add', elementController.addElement);
 
 router
     .route('/:id')
+    .get(elementController.getElement)
```
Comment (K-do) [inline Server/routes/elementRoutes.js:32]: You don't want to allow non-admin users to get an Element ?

File: Server/test/controller/authControllerTest.js [hunk 2]
```diff
@@ -86,6 +85,7 @@ describe('AuthControler', function () {
                     password: "012345678",
                     role: "admin"
                 });
+            admin = res;
```
Comment (K-do) [inline Server/test/controller/authControllerTest.js:88]: never used

File: Server/test/controller/authControllerTest.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ const {ConnectionEvent} = require("../../models/eventModel");
 const timeoutDuration = 3000;
 
 let user;
+let admin;
```
Comment (K-do) [inline Server/test/controller/authControllerTest.js:21]: never used

File: Server/test/models/QRModelTest.js [hunk 4]
```diff
@@ -60,15 +59,20 @@ describe('QRModel', function () {
...
                 expect(data.code).to.be.equal(value.toString());
                 expect(data.id).to.be.not.empty;
             });
         });
     });
 });
 
+after(async function () {
+    await QR.deleteMany({
+        code:value
```
Comment (K-do) [inline Server/test/models/QRModelTest.js:75]: need blanc between "code:vlue" -> "code: value"

**Review by K-do — CHANGES_REQUESTED** [review]
Respond to comments and i'll approve

File: Server/routes/elementRoutes.js (hunk 1 above)
Comment (LeonardBesseau) [reply Server/routes/elementRoutes.js:32]: Admin check is after

File: Server/test/controller/authControllerTest.js (hunk 2 above)
Comment (LeonardBesseau) [reply Server/test/controller/authControllerTest.js:88]: It is but as a global. Not really pretty but it speeds up the test by 500 ms so why not

File: Server/test/controller/authControllerTest.js (hunk 3 above)
Comment (LeonardBesseau) [reply Server/test/controller/authControllerTest.js:21]: Same one as before.

**Comment by LeonardBesseau** [comment]
Done. Strangely one test was failing and then started working again. All good now

File: Server/test/controller/QRControllerTest.js [hunk 5]
```diff
@@ -25,6 +25,7 @@ let idManager;
 let idQR1;
 let idQR2;
 let idQR3;
+let idRandom;
```
Comment (K-do) [inline Server/test/controller/QRControllerTest.js:28]: ?

**Review by K-do — APPROVED** [review]

**Pull request merged**